docs/events/api/timeline-api-get.asciidoc

```diff
@@ -1,86 +1,660 @@
 [[timeline-api-get]]
 === Get Timelines or Timeline templates
 
-Get Timelines or Timeline templates.
+Retrieves a list of all Timelines or Timeline templates.
 
 ==== Request URL
 
 `GET <kibana host>:<port>/api/timelines`
 
-==== Request params
+==== URL query parameters
 
 [width="100%",options="header"]
 |==============================================
 |Name |Type |Description |Required
 
-|`only_user_favorite` |Boolean a|Returns only favorite Timelines / Timeline templates if set to `true`. Default to `false`
+|`only_user_favorite` |Boolean a|If set to `true`, returns only favorite Timelines / Timeline templates. Defaults to `false`.
 |No
-|`page_index` |Number |Page number of the result
+|`page_index` |Number |The page number to return. Defaults to TBD.
```

Comment:
@elastic/security-threat-hunting-investigations, could you provide the default values for `page_index` and ` page_size`? Thanks!

Reply:
This should be page 1 as it's requesting the full list of timelines and will return the first page by default